.gitignore

```diff
@@ -23,3 +23,4 @@ project/metals.sbt
 .scalafmt-common.conf
 .scalafix-common.conf
 .vscode
+/\.bsp
```

Comment:
Can we add a new line here please? 
Also why /\? Do we need to escape characters in .gitignore? 🤔

Reply:
Why do we need newline ?
and im not sure this was copied from somwehere else will investigate, if it is a regex then it would need escaping